dist/jquery.mark.es6.js

```diff
@@ -70,7 +70,7 @@
                 return;
             }
             if (typeof log === "object" && typeof log[level] === "function") {
-                log[level](`mark.js: ${msg}`);
+                log[level](`mark.js: ${ msg }`);
```

Comment:
Any idea why this happens? It seems like every time I run `grunt dist` these placeholders alternate between `${ x }` and `${x}`.